pipeline/bicleaner/bicleaner.sh

```diff
@@ -75,6 +75,10 @@ else
        biclean() {
                export CUDA_VISIBLE_ARRAY=(${CUDA_VISIBLE_DEVICES//,/ })
                export CUDA_VISIBLE_DEVICES=${CUDA_VISIBLE_ARRAY[$(($2-1))]}
+               # The GPU devices have failed to be found, and bicleaner AI falls back
+               # to operate on the CPU very slowly. To guard against this wasting expensive
+               # GPU time, always check that it can find GPUs.
+               python -c "import tensorflow; exit(0) if tensorflow.config.list_physical_devices('GPU') else exit(9001)"
```

Comment:
I think you might need to use `python3` here to choose the correct version. It's what we usually do for python scripts in Taskcluster tasks.